build/UIKit/dll/UIKit.def

```diff
@@ -1173,3 +1173,9 @@ LIBRARY UIKit
         UIOrientationFromString
         UIApplicationDidChangeDisplayModeNofication
         UIViewControllerFirePageEvent
+
+        ;_UIGestureCoordinator.mm 
+        _OBJC_CLASS__UIGestureCoordinator DATA
+        __objc_class_name__UIGestureCoordinator CONSTANT
+
+        
```

Comment:
ah - i see you have two files with the same name.  maybe this export is needed after all..

---
In reply to: [100644840](https://github.com/Microsoft/WinObjC/pull/1909#discussion_r100644840) [](ancestors = 100644840)